src/index.html

```diff
@@ -15,8 +15,89 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#src\images\logo.png">
+        <img
+          src="images/logo.png"
+          alt="logo_moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li>
```

Comment:
Add classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```